#This message is an array of grasp messages
Header header
jarvis_perception/GraspBox[] grasps
